packages/salesforcedx-vscode-apex/src/languageClientUtils/languageClientUtils.ts

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { LanguageClient } from 'vscode-languageclient';
+import {
+  DEBUGGER_EXCEPTION_BREAKPOINTS,
+  DEBUGGER_LINE_BREAKPOINTS
+} from '../constants';
+import {
+  ApexLSPConverter,
+  ApexTestMethod,
+  LSPApexTestMethod
+} from '../views/lspConverter';
+
+export class LanguageClientUtils {
```

Comment:
I wanted `LanguageClientUtils` to only be modified by the `salesforcedx-vscode-apex` extension and then have the debugger extensions exclusively consume the status of the Apex language server (`LanguageClientStatus`). This is why I only expose `languageClientUtils.getStatus()` as part of the extensions api.